todo/services/label_service.py

```
@@ -0,0 +1,91 @@
+            prev_link = cls.build_page_url(prev_page, limit, search)
+
+        return LinksData(next=next_link, prev=prev_link)
+
+    @classmethod
+    def build_page_url(cls, page: int, limit: int, search: str) -> str:
+        base_url = reverse_lazy("labels")
+        query_params = urlencode({"page": page, "limit": limit, "search": search})
+        return f"{base_url}?{query_params}"
+
+    @classmethod
+    def prepare_label_dto(cls, label_model: LabelModel) -> LabelDTO:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)